styleguide.config.js

```diff
@@ -92,6 +92,21 @@ module.exports = async () => {
 			{
 				name: 'Directives',
 				content: 'docs/directives.md',
+				sectionDepth: 1,
+				sections: [
+					{
+						name: 'Focus',
+						content: 'docs/directives/focus.md',
+					},
+					{
+						name: 'Linkify',
```

Comment:
I'd love to see component documentations in separate files as well, to keep SFC clean, but something for later maybe

Reply:
Or at least move it to the end of the file, not the top 🥲